app/scripts/controllers/membership.js

```diff
@@ -88,16 +91,16 @@ angular
           .then(function() {
             resetForm();
             refreshRoleBindingList();
-            showAlert('rolebindingCreate', 'success', messages.update.subject.success({
+            showAlert('rolebindingCreate', 'success', escaped.update.subject.success({
```

Comment:
It sounds like they strip things like `<script>` tags from the content, which is good. But if you have a user name with special characters like `<` and `&`, I suspect it won't render correctly.